Test/Framework/Visual/VertexBufferTest.cs

```diff
@@ -121,7 +121,36 @@ public void GetPosition(bool dynamic)
                 Assert.AreEqual(savedData[2].Position, readData[2]);
                 Assert.AreEqual(savedData[3].Position, readData[3]);
             };
-            Game.RunOneFrame();
+            Game.Run();
+        }
+
+        //[TestCase(true)]
```

Comment:
> although these tests also use `GetData`, which isn't implemented at all for dynamic vertex buffers 

We can maybe implement `GetData`... although I don't want to encourage people to use it on any regular basis.

Another approach is to validate the results by rendering something.